projects/05/Memory.hdl

```diff
@@ -28,7 +28,8 @@ CHIP Memory {
 
     PARTS:
 
-    DMux(in=load, sel=address[14], a=loadRAM, b=loadScreen);
+    DMux(in=load, sel=address[14], a=loadRAM, b=loadScreenOrKeyboard);
+    DMux(in=loadScreenOrKeyboard, sel=address[13], a=nothing, b=loadScreen);

```

Comment:
There’s a problem here. What do the different values of `address[13]` mean?

Reply:
Oh, this is a bug!! If address[13] = 0, means we load the screen. If address[13] = 1 then we "load" the keyboard. It seems like I've misplaced the `a` and `b` output for the `DMux`. If address[13] is 0, we want the `a` output to be for the screen, not the keyboard.

Reply:
Perfect! You nailed it 🎉